### PR TITLE
Added new 'Bot Accounts' page under 'Integrations'

### PR DESCRIPTION
**Pull request by aaronrothschild**
-Moved EnableBotCreation from previous location
- Added new settings `DisableBotsWhenOwnerIsDeactivated`

https://github.com/mattermost/mattermost-webapp/pull/2954

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -2745,6 +2745,29 @@ To manage who can create personal access tokens or to search users by token ID,
 | This feature's ``config.json`` setting is ``"EnableUserAccessTokens": false`` with options ``true`` and ``false``.                                                   |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+
+Bot Accounts
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Enable Bot Account Creation
```
Comment (amyblais) [inline source/administration/config-settings.rst]: @aaronrothschild Should this setting be removed from "Integrations" section above or does it still also exist there in the system console?

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -2745,6 +2745,29 @@ To manage who can create personal access tokens or to search users by token ID,
...
+| This feature's ``config.json`` setting is ``"EnableBotAccountCreation": false`` with options ``true`` and ``false``.                                                 |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Disable bot accounts when owner is deactivated
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+**True**: When a user is deactivated, disables all bot accounts managed by the user. To re-enable bot accounts, go to **Integrations > Bot Accounts**.
+
+**False**: When a user is deactivated, all bot accounts managed by the user remain active. 
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (amyblais) [inline source/administration/config-settings.rst]: This formatting is broken. Please choose "No wrap" to format the table correctly.

File: source/administration/config-settings.rst (hunk 1 above)
Comment (aaronrothschild) [reply source/administration/config-settings.rst]: On 5.14 it no longer appears in the "Integrations" page

Comment (amyblais) [reply source/administration/config-settings.rst]: Cool, then it can be removed from that section.

**Pull request merged**